tests/cases/fourslash/completionListInObjectLiteral7.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path='fourslash.ts'/>
+
+////type Foo = { foo: boolean };
+////function f<T>(shape: Foo): any;
+////function f<T>(shape: () => Foo): any;
+////function f(arg: any) {
+////  return arg;
+////}
+////
+////f({ /*1*/ });
+////f(() => ({ /*2*/ }));
+////f(() => (({ /*3*/ })));
```

Comment:
related to my other comment, do parenthesised object binding assignments still fail? If it is syntactically correct, I'd like to see a test for that too.

Reply:
I think that we cannot use `()` in object binding assignments

```ts
let x;
(({ x }) = { x: 1 })
  ^^^^^ Uncaught SyntaxError: Invalid left-hand side in assignment
```

Reply:
@sandersn Do you have an example that I can use in a test?

Reply:
You gave the one I was thinking about. If it's illegal then no need for a test.